microplan-plan.js

```diff
@@ -12,5 +15,13 @@ if (!args.length) {
 
 if (program.force) console.log('  force: install')
 args.forEach(function (arg) {
+  // Get document, or throw exception on error
+  try {
+    var doc = yaml.safeLoad(fs.readFileSync(path.join(__dirname, arg), 'utf8'))
```

Comment:
there is a high chance that we refactor this in future. because , we will use the parsing module in future.